yt_dlp/extractor/radiocomercial.py

```diff
@@ -0,0 +1,150 @@
+import itertools
+
+from .common import InfoExtractor
+from ..networking.exceptions import HTTPError
+from ..utils import (
+    ExtractorError,
+    extract_attributes,
+    get_element_by_class,
+    get_element_html_by_class,
+    get_element_text_and_html_by_tag,
+    get_elements_html_by_class,
+    int_or_none,
+    join_nonempty,
+    try_call,
+    unified_strdate,
+    update_url,
+    urljoin
+)
+from ..utils.traversal import traverse_obj
+
+
+class RadioComercialIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?radiocomercial\.pt/podcasts/[^/?#]+/t?(?P<season>\d+)/(?P<id>[\w-]+)/?(?:$|[?#])'
```

Comment:
Read https://github.com/yt-dlp/yt-dlp/pull/8535#discussion_r1390220122
Here I have the same question:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?radiocomercial\.pt/podcasts/[^/?#]+/t?(?P<season>\d+)/(?P<id>[\w-]+)'
```
It's needed in `RadioComercialPlaylistIE._VALID_URL`, but I don't think we need it here? Unless I'm missing something

Reply:
I believe you are correct bashonly. I removed it from the single episode regex.